.github/workflows/composite/setup/action.yaml

```diff
@@ -25,13 +26,13 @@ runs:
     - name: Download MetalLB images
       uses: actions/download-artifact@v3
       with:
-        name: metallb-images
         path: metallb-images
 
     - name: Load MetalLB images
       shell: bash
       working-directory: metallb-images
       run: |
-        docker load -i speaker.tar
-        docker load -i controller.tar
-        docker load -i mirror-server.tar
+        docker load -i image-tar-speaker/speaker.tar
+        docker load -i image-tar-controller/controller.tar
+        docker load -i image-tar-mirror-server/mirror-server.tar
+        docker load -i image-tar-configmaptocrs/configmaptocrs.tar
```

Comment:
I don't think we use this anywhere, we can skip this? @fedepaol

Reply:
I think we do

Reply:
They are used in the e2e testing, if i´m not mistaken. You load the images so when you do you´re deploying, you´re loading the pipelines image

Reply:
Not the `mirror-server` image I think

Reply:
If mirror-server is not used in testing and not published, where is it used? These loads are only for testing.

Reply:
It's what we discussed here https://github.com/metallb/metallb/pull/1414#discussion_r892413347 but I wasn't totally sure so we agreed to keep it for now.
I don't think it is used at all, so if you want you can remove everything mirror-server related in this PR (in a separate commit) or we can handle it later

Reply:
I would say for now, to keep things atomic and easier to comprehend, lets just leave it and remove in another pr.
(ie this mr changes everything to a different method, another mr removes unneeded components)

Reply:
Agree

Reply:
I have a separate MR ready for it, #1436 , so I dont forget.